Changes

```diff
@@ -2,6 +2,41 @@
 Revision history for Perl extension Mojo-IOLoop-ReadWriteProcess
 
 {{$NEXT}}
+1.0.0 2025-03-17 12:20:20Z
+ -  CI: Bump to codecov-action@v5
+ -  CI: Use codecov token to prevent rate limit
+ -  CI: Bump to codecov-action@v4
+ -  CI: Fix indendation in version matrix
+ -  CI: Simply use version 'latest' for codecov upload
+ -  CI: Remove trailing space in title
+ -  CI: Simplify 'on' definition
+ -  CI: Add recent and current Perl versions
+ -  CI: Use current version of checkout@v4
+ -  Remove .proverc
+ -  t: Use $^X for executing perl
+ -  Update condition for process list definitions in 12_mocked_container
+ -  Use consistent interval in `t/01_run.t`
+ -  Fix more instances where `t/01_run.t` might fail
+ -  Give process group a few seconds to terminate in `t/01_run.t`
+ -  Fix sporadic failures in `t/01_run.t`
+ -  Debug test for stopping whole process group
+ -  Fix race condition in `is_running` when `kill_whole_group` is set
+ -  Apply tidy 20250105
+ -  Fix handling process groups when initial process is not running anymore
+ -  Use the correct formatter
+ -  Fix errors on Build.PL
+ -  Remove workaround for missing dependency on TAP::Harness::GithubActions
+ -  Update to codecov@v3
+ -  Add Minilla to the ci target
+ -  Update minil.toml to have the harness arguments enabled
+ -  Use github formatter for prove
+ -  Fix tests and rename workflow
+ -  Use reduced test matrix on MacOSX
+ -  Test on OSX and Linux with a reduced perl matrix
+ -  t: Add another check for /bin/true
+ -  Fix tidy also for OSX
+ -  Mention syntactic sugar to help when debugging
+ -  Enable support for MacOSX (darwin)
```

Comment:
Too long for a user facing Changelog.
Especially CI updates aren't of any interest.

Reply:
```suggestion
 - Various test fixes
 -  Fix race condition in `is_running` when `kill_whole_group` is set
 -  Fix handling process groups when initial process is not running anymore
 -  Add Minilla to the ci target
 -  Update minil.toml to have the harness arguments enabled
 -  Mention syntactic sugar to help when debugging
 -  Enable support for MacOSX (darwin)
```